code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingConstants.java

```diff
@@ -407,4 +407,26 @@ public class PayloadKeys {
             private PayloadKeys() {}
         }
     }
+
+    public final class UIKeys {
```

Comment:
Messaging extension has other UIs as well: Push, IAM. These are specific to content cards. Can you add this inside `public final class ContentCard`